src/pages/landing/Landing.tsx

```diff
@@ -0,0 +1,201 @@
+              ${tier.usdPrice.toFixed(2)}
+            </Typography>
+            <Typography variant="body1" className="font-bold">
+              {tier.subtitle}
+            </Typography>
+            <Typography variant="body1" className="font-normal mb-12">
+              {tier.features.map(feature => (
+                <span key={feature.name}>{feature.name}<br/></span>
+              ))}
+            </Typography>
+            <Button
+              variant="contained"
+              size="large"
+              className="py-3 px-14"
+              color={!props.tierId || props.tierId === tier.id ? 'info' : 'primary'}
+              onClick={() => props.onSelect(tier.id)}
+            >
+              {props.tierId === tier.id ? 'Selected' : 'Select'}
+            </Button>
+          </div>
+        ))}
+      </Box>
+    </Box>
+  </Box>
+);
+
+const RegisterSection = (props: { tierId: string, count: number, price: number, onSubmit: (event: React.SyntheticEvent) => void }) => (
+  <Box className="flex flex-col m-0 pt-24 min-w-[50vw] bg-white text-center">
+    <Typography variant="h4" className="font-bold mb-2 tracking-[.2em] capitalize">
+      Auditor profile
+    </Typography>
+    <Typography variant="subtitle1" className="font-normal mb-16 text-[#A6A7AD]">
+      Please complete the information to create your account.
+    </Typography>
+    <Container maxWidth="sm">
+      <form onSubmit={props.onSubmit}>
+        <TextField
+          className="mb-6"
+          required fullWidth variant="standard"
+          label="Company name" name="company" type="text"
+        />
+        <TextField
+          className="mb-6"
+          required fullWidth variant="standard"
+          label="Email" name="email" type="email"
+        />
+        <TextField
+          className="mb-6"
+          required fullWidth variant="standard"
+          label="Address" name="address" type="text"
+        />
+        <TextField
+          className="mb-6"
+          required fullWidth variant="standard"
+          label="City, zip code" name="city" type="text"
+        />
+        <TextField
+          className="mb-6"
+          required fullWidth variant="standard"
+          label="Country" name="country" type="text"
+        />
+        <TextField
+          className="mb-6"
+          fullWidth variant="standard"
+          label="Twitter" name="twitter" type="text"
+        />
+        <TextField
+          className="mb-6"
+          fullWidth variant="standard"
+          label="LinkedIn" name="linkedin" type="text"
+        />
+        <Button
+          type="submit"
+          variant="contained"
+          size="large"
+          className="mt-8 py-3 px-14 normal-case"
+        >
+          Pay (₳{props.price.toFixed(2)})
+        </Button>
+        <Typography variant="body1" className="font-normal text-sm mt-6 text-red-500">
+          Price will refresh in {props.count} seconds
```

Comment:
I'm nit picking, don't do it if it takes more than 5 seconds to fix.

Can we have a function like this one:
```
function s_of_int(n: number): string {
  return n !== 1 ? "s" : "";
}
```
so that it displays "Price will refresh in 1 second" instead of "1 seconds"?

Reply:
Sure!